<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>isoline_dialog</class>
 <widget class="QDialog" name="isoline_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>411</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>425</width>
    <height>317</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Orbitals</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Orbital type:</string>
   </property>
  </widget>
  <widget class="QLabel" name="orbTypeLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>101</width>
     <height>16</height>
    </size>
   </property>
   <property name="text">
    <string>Cartesian</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Orbital:</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="orbitalSelector">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>191</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Spin:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="alphaCheckBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Alpha</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="betaCheckBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Beta</string>
   </property>
  </widget>
  <widget class="QLabel" name="restrictedLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Restricted</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>260</y>
     <width>211</width>
     <height>141</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Exact Electroncount:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculated Electroncount:</string>
    </property>
   </widget>
   <widget class="QLabel" name="exactCountLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="normalizationButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>190</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Check Normalization</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="calculatedCountLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="displayMos">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>190</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Display Orbital</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>211</width>
     <height>91</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Color -</string>
    </property>
   </widget>
   <widget class="QPushButton" name="colorPlusButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Color +</string>
    </property>
   </widget>
   <widget class="QPushButton" name="colorMinusButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
